golf2/src/digest.rs

```diff
@@ -53,9 +53,11 @@ impl<'a, E: DigestEngine + 'a> DigestDriver<'a, E> {
 
 impl<'a, E: DigestEngine> Driver for DigestDriver<'a, E> {
     fn command(&self, minor_num: usize, r2: usize, _r3: usize, caller_id: AppId) -> ReturnCode {
+        //print!("DigestEngine: command({}, {}, {})\n", minor_num, r2, _r3);
```

Comment:
Line 56 appears to be a leftover debugging statement -- please remove.

Reply:
Done.